src/app/components/leftbar/leftbar.component.html

```diff
@@ -1,18 +1,18 @@
 <div class="ccf-leftbar-wrapper">
   <div class="container">
     <div class="leftbar">
-      <div class="search">
-        <ccf-search-icon (click)="emitToggleEvent()"></ccf-search-icon>
+      <div [ngClass]="{'search': true, 'search-expanded': sidenavExpanded}">
+        <ccf-search-icon (click)="emitToggleEvent()" ></ccf-search-icon>
       </div>
       <mat-list class="breadcrumb">
-        <mat-list-item>
-          <ccf-body-icon></ccf-body-icon>
+        <mat-list-item class="breadcrumb-item">
```

Comment:
No need to prefix with 'breadcrumb' use the class 'item' instead. Scss nesting make prefixing unnecessary. Same goes for the icons.